metaflow/plugins/kfp/tests/run_integration_tests.py

```diff
@@ -0,0 +1,71 @@
+from os import listdir
+from os.path import isfile, join
+from subprocess import run, PIPE
+from typing import List
+
+from .... import R
+
+import kfp
+
+import pytest
+
+"""
+From the folder `tests`, run: 
+`python -m pytest -s -n 3 run_integration_tests.py`
+
+This script runs all the flows in the `sample_flows` directory. It creates
+each kfp run, waits for the run to fully complete, and prints whether
+or not the run was successful. It also checks to make sure the logging
+functionality works.
+
+More specifically, the tests spawn KFP runs and ensure the spawning processes
+have a returncode of 0. However, the logs of the runs within KFP are simply logged
+to the screen, and it is the responsibility of the user to read these logs to ensure
+the run executed correctly on KFP.
```

Comment:
It is the responsibility of the Flow to ensure that the run succeeded, via assert and test validation checks at the end. 

It is NOT the responsibility of the user to read output logs, this needs to be automated.

Reply:
My bad, this comment is outdated. The test raises an exception is any of the flows fail. Will delete that part of the comment.